gtest_parallel.py

```diff
@@ -454,7 +454,7 @@ def find_tests(binaries, additional_args, options, times):
 
       test_count += 1
 
-  return tasks
+  return sorted(tasks)
```

Comment:
(Leaving comment here instead of on `Task.__lt__`, since Github won't let me add a comment on a line too far away from the change. Irritating...)

Looks good, except that Task implements `__lt__` but not the other comparison special functions, and the implementation of `__lt__` says that a < b _and_ b < a if a and b both have no last_execution_time. I'd suggest replacing it with
```
def __sorting_key(self):
  return (0 if self.last_execution_time is None else 1,
          self.last_execution_time,
          ...all the other fields here is some sane order...)

def __cmp__(self, other):
  return cmp(self.__sorting_key(), other.__sorting_key())
```
which solves both problems.